packages/dx-react-grid/docs/reference/paging-state.md

```diff
@@ -16,7 +16,7 @@ totalCount | number | | Specifies the total rows count
 currentPage | number | | Specifies the current page number
 defaultCurrentPage | number | 0 | Specifies the initial current page for the uncontrolled mode
 onCurrentPageChange | (currentPage: number) => void | | Handles current page changes
-pageSize | number &#124; | | Specifies the page size
+pageSize | number | | Specifies the page size
```

Comment:
Yes, I saw it. I think that reference is a more suitable place for this. It is also good to have a note about 0 constant in guide and reference.